examples/apollo-federation-compatibility/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "graphql-yoga-federation-service",
+  "version": "1.0.0",
+  "description": "Federation Service implemented with GraphQL Yoga",
+  "main": "index.js",
+  "scripts": {
+    "prebuild": "graphql-codegen",
+    "build": "npx tsc",
+    "start": "node ./dist/index.js"
+  },
+  "author": "Charly POLY",
```

Comment:
@enisdenjo 👀

Reply:
I copied most of @charlypoly's work from https://github.com/apollographql/apollo-federation-subgraph-compatibility/tree/main/implementations/graphql-yoga, he's the real author. 😄